providers/implementations/keymgmt/dh_kmgmt.c

```diff
@@ -208,6 +208,8 @@ static int dh_export(void *keydata, int selection, OSSL_CALLBACK *param_cb,
     return ok;
 }
 
+/* IMEXPORT = IM- + EXPORT */
```

Comment:
= IMPORT + EXPORT

Reply:
Hmm?  That hyphenated form is unknown in English?  We do that all the time in Swedish...

Reply:
Changed